app/src/main/java/ru/yandex/practicum/contacts/presentation/base/ListDiffInterface.java

```diff
@@ -0,0 +1,6 @@
+package ru.yandex.practicum.contacts.presentation.base;
+
+public interface ListDiffInterface <T> {
+    boolean theSameAs(T o);
```

Comment:
🍏 Рекомендую давать переменным/параметрам/аргументам развернутые названия.
Например, 
```suggestion
    boolean theSameAs(T newItem);
```